packages/quantic/force-app/main/default/lwc/quanticSort/e2e/sortObject.ts

```diff
@@ -0,0 +1,76 @@
+import {Locator, Page, Request} from '@playwright/test';
+import {isUaSearchEvent} from '../../../../../../playwright/utils/requests';
+
+export class SortObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get sortDropDown(): Locator {
+    return this.page.getByRole('combobox', {name: 'Sort by'});
+  }
+
+  get sortPreviewHeader(): Locator {
+    return this.page.getByRole('button', {name: 'Preview'});
+  }
+
+  get invalidMessage(): Locator {
+    return this.page
+      .locator('p')
+      .filter({hasText: 'Custom sort options configuration is invalid.'});
+  }
+
+  sortButton(buttonName: string): Locator {
+    return this.page.getByRole('option', {name: buttonName});
+  }
+
+  async clickSortDropDown(): Promise<void> {
+    await this.sortDropDown.click();
+  }
+
+  async focusSortDropDownEnter(): Promise<void> {
+    await this.sortPreviewHeader.click();
+    await this.page.keyboard.press('Tab');
+  }
+
+  async clickSortButton(buttonName: string): Promise<void> {
+    await this.sortButton(buttonName).click();
+  }
+
+  async selectSortButtonKeyboard(): Promise<void> {
```

Comment:
In line with my suggestion to test with space, you could probably modify this to take the key you want to use as argument (ENTER || SPACE)